tests/sys_sched_round_robin/Makefile

```diff
@@ -0,0 +1,5 @@
+include ../Makefile.tests_common
```

Comment:
Please drop the `thanks to @maribu for describing this test by deadlock` from the commit message.

Having @ mentions in commit messages results in receiving a lot of notifications... I did this once and now receive random notifications from other RIOT forks...

Reply:
done